test/yb-ctl-test.sh

```diff
@@ -94,6 +94,8 @@ start_cluster_run_tests() {
 }
 
 readonly yb_data_dir="/tmp/yb-ctl-test-data-$( date +%Y-%m-%dT%H_%M_%S )-$RANDOM"
+readonly yb_univ_1_data_dir="/tmp/yb-ctl-test-data-$( date +%Y-%m-%dT%H_%M_%S )-$RANDOM-1"
+readonly yb_univ_2_data_dir="/tmp/yb-ctl-test-data-$( date +%Y-%m-%dT%H_%M_%S )-$RANDOM-2"
```

Comment:
Maybe create a common parent directory for the three data directories?
E.g.

```bash
readonly yb_data_dir_parent="$( date +%Y-%m-%dT%H_%M_%S )-$RANDOM"
readonly yb_data_dir=$yb_data_dir_parent/single_univ
readonly yb_univ_1_data_dir=$yb_data_dir_parent/cdc_1
readonly yb_univ_2_data_dir=$yb_data_dir_parent/cdc_2
```

Reply:
Done.